3-extra/2-sorting-algorithm.js

```diff
@@ -14,7 +14,25 @@ You don't have to worry about making this algorithm work fast! The idea is to ge
 "think" like a computer and practice your knowledge of basic JavaScript.
 */
 
-function sortAges(arr) {}
+function sortAges(arr) {
+ let resultArr = [];
+
+  for (let i = 0; i < arr.length; i++) {
+    if (/[0-9]/.test(arr[i])) {
```

Comment:
This `.test` will return true if the item is a string containing at least one number. But we want numbers not strings. Could try `Number.isInteger()`?